datasquare_test.go

```diff
@@ -420,6 +422,54 @@ func BenchmarkEDSRoots(b *testing.B) {
 	}
 }
 
+func BenchmarkEDSRootsWithErasuredNMT(b *testing.B) {
+	ODSSizeByteUpperBound := 512 * 1024 * 1024 // converting 512 MiB to bytes
```

Comment:
[optional] define a constant

```go
const mebibyte = 1_048_576 // bytes
```

then we don't need a comment here

```go
ODSSizeByteUpperBound := 512 * mebibyte
```

Reply:
I applied your suggestion [a67997e](https://github.com/celestiaorg/rsmt2d/pull/304/commits/a67997ec08f9079f99271fd78dbb84af4bea74f2)